share/lib/python/neuron/rxd/node.py

```diff
@@ -92,6 +94,12 @@ def _replace(old_offset, old_nseg, new_offset, new_nseg):
         _states[new_offset + j ] = _states[old_offset + i]
     _states = numpy.delete(_states,list(range(start, stop)))
 
+    # update _node_flux index
+    for (i,idx) in enumerate(_node_fluxes['index']):
+        if idx in dels:
```

Comment:
it is not very efficient to look in `dels` when you can simply compare `start <= idx < stop`